proto/v3/message_contents/message.proto

```diff
@@ -47,15 +56,26 @@ enum PadlockMessagePayloadVersion {
 //    sender_installation_id
 // 5. Verify that both the sender_user and recipient_user are partipants of the
 //    conversation referenced by convo_id
-message PadlockMessagePayload {
-  PadlockMessagePayloadVersion message_version = 1;
-  EdDsaSignature header_signature = 2; // Signs PadlockMessageHeader
-  string convo_id = 3;
-  bytes content_bytes = 4; // EncodedContent
+message MessagePayloadV1 {
+  EdDsaSignature header_signature = 1; // Signature of the header_bytes
```

Comment:
Depending on the details of how we are implementing sealedsender, I'm wondering if we still need to sign this